pkg/ds/daemon_set.go

```diff
@@ -161,8 +150,17 @@ func (ds *daemonSet) WatchDesires(
 					continue
 				}
 
-			case <-deletedCh:
-				ds.logger.NoFields().Infof("Received daemon set delete signal")
+			case deleteDS := <-deletedCh:
```

Comment:
The current implementation `WatchDiff` will allow the caller to know what was deleted because it keeps a copy of the daemon set in memory. Since the farm uses `WatchDiff` and `WatchDesires`, it will know the contents.